executor/executor_test.go

```diff
@@ -4248,6 +4248,7 @@ func (s *testSplitTable) TestShowTableRegion(c *C) {
 	tk := testkit.NewTestKit(c, s.store)
 	tk.MustExec("use test")
 	tk.MustExec("drop table if exists t_regions")
+	tk.MustExec("set global tidb_scatter_region = 1")
```

Comment:
Why it can fix the issue of #17189？
```
FAIL: executor_test.go:4247: testSplitTable.TestShowTableRegion

executor_test.go:4260:
    // Table t_regions should have 5 regions now.
    // 4 regions to store record data.
    // 1 region to store index data.
    c.Assert(len(rows), Equals, 5)
... obtained int = 6
... expected int = 5
```

Reply:
Sorry, I add a description in PR, this pr is not fixed this issue  #17189